database/database-cft.yaml

```diff
@@ -0,0 +1,107 @@
+AWSTemplateFormatVersion: 2010-09-09
+Description: Template for creating testbed database instance
+
+Parameters:
+  StaticStackName:
+    Type: String
+    Default: dev-ga4gh-testbed-database
+  ParamDBName:
+    Type: String
+    Default: testbed_db
+  ParamUsername:
+    Type: String
+    Default: testbed_writer
+  SubnetA:
+    Type: String
+    Default: subnet-0be7a1fb3a96191b4
+  SubnetB:
+    Type: String
+    Default: subnet-0c465ec61b98bffac
+  SubnetC:
+    Type: String
+    Default: subnet-07b0e79f4d9bbf9d0
+
+Resources:
+
+  WebServerSecurityGroup:
+    Type: AWS::EC2::SecurityGroup
+    Properties:
+      GroupDescription: Enable HTTP and SSH connection
+      GroupName: TestBedServerSecurityGroup
+      SecurityGroupIngress:
+        - IpProtocol: tcp
+          FromPort: 4500
+          ToPort: 4500
+          CidrIp: 0.0.0.0/0
+        - IpProtocol: tcp
+          FromPort: 80
+          ToPort: 80
+          CidrIp: 0.0.0.0/0
+        - IpProtocol: tcp
+          FromPort: 22
+          ToPort: 22
+          CidrIp: 0.0.0.0/0
+        - IpProtocol: tcp
+          FromPort: 5432
+          ToPort: 5432
+          CidrIp: 0.0.0.0/0
+
+  ApiSecret:
```

Comment:
Do you plan on using this resource? If not, remove this

Reply:
Removed.